zbus/src/match_rule/mod.rs

```diff
@@ -21,7 +21,7 @@ pub use builder::Builder;
 
 /// A bus match rule for subscribing to specific messages.
 ///
-/// This is mainly used by peer to subscribe to specific signals as by default the bus will not
+/// This is mainly used for p2p to subscribe to specific signals as by default the bus will not
```

Comment:
This is incorrect. We really mean a peer here. A match-rule is mainly for the bus case.

Reply:
Right, the wording "by peer" didn't make sense to me so I assumed this must be talking about p2p. What do peers have to do with things if it's between us and the bus?

For now, I've removed the words entirely ("This is mainly used to subscribe"), since this reads best to me. Let me know if you'd prefer something else.

Reply:
`peer` is a term used in the D-Bus world frequently for the bus clients, perhaps cause it can get really confusing otherwise if services are also clients.

> For now, I've removed the words entirely ("This is mainly used to subscribe"), since this reads best to me. Let me know if you'd prefer something else.

That sounds good to me. Maybe it will be confusing for others too.